static/app/views/performance/table.tsx

```diff
@@ -456,6 +454,16 @@ class _Table extends Component<Props, State> {
       />
     );
     if (field.field.startsWith('user_misery')) {
+      if (title.tooltip) {
+        title.tooltip ? <Tooltip isHoverable title={title.tooltip} showUnderline /> : '';
```

Comment:
🤔 Is this line doing anything?

Reply:
whoops, I accidentally left that in good catch.